### ISSUE TITLE
Intro link on top page of each lecture series is out of data

### ISSUE THREAD

**Issue by jstac** [QuantEcon/meta#113]
Most lecture series in the existing and reorganized lectures contain the line "For an overview of the series, see [this page](https://quantecon.org/python-lectures/)".  

The page that it points to is outdated and should be retired.

I suggest we replace them all with: "A list of all lecture series can be found [here](https://quantecon.org/projects/#filter=lecture)"

**Comment by jstac** [comment]
@mmcky , thanks for addressing this quickly.

Should we leave it open while we deal with https://github.com/orgs/QuantEcon/discussions/111 ?

(It's referenced there.)

I'll reopen but feel free to close if you don't need it.

**Comment by mmcky** [comment]
Yeah sorry -- I used the `fixes` and that closed this automatically. I agree this should remain open.

**Comment by HumphreyYang** [comment]
Many thanks @jstac and @mmcky,

All my lecture migrations are automatic on my side (a very long migration Python code). As long as the changes are in the main branch for corresponding lectures, they will be added to the migrated series after I run the program on my side.

**Comment by jstac** [comment]
Roger that, thanks @HumphreyYang

**Comment by mmcky** [comment]
@HumphreyYang the changes to fix this short term was to fix the `redirect` on the web site of things. We will still need to go through and update links etc. so we don't have to rely on the redirect

**Comment by HumphreyYang** [comment]
> @HumphreyYang the changes to fix this short term was to fix the `redirect` on the web site of things. We will still need to go through and update links etc. so we don't have to rely on the redirect

Many thanks @mmcky. I think we can have long-term patches once we reorganize the lectures. Could we take this into account when designing the opening statement for each series if that's possible?
